CMakeLists_files.cmake

```diff
@@ -478,6 +478,7 @@ if(ENABLE_ECL_INPUT)
     examples/opmpack.cpp
     examples/opmhash.cpp
     examples/wellgraph.cpp
+    examples/make_lodsmry.cpp
```

Comment:
You **may** want to add `make_lodsmry` to the [`PROGRAM_SOURCE_FILES`](https://github.com/OPM/opm-common/blob/0756cd93ada466a7b96dfbe34d33103ab94981af/CMakeLists_files.cmake#L489-L493) too.  That way, it'll be copied out to the final destination during `make install` and also available in the packaged binaries.  As of right now, the `make_lodsmry` binary will only be available in the build tree, meaning a prospective user needs to build OPM-Core just to get the helpful program.

Reply:
Thanks, I have done that now.

Reply:
> Thanks, I have done that now.

Thanks a lot.  Let's just run a final build check and then I'm ready to merge this.  The broader discussion about `LODSMRY` and HDF5 can wait until a later date.